invenio_administration/templates/semantic-ui/invenio_administration/index.html

```diff
@@ -1,8 +1,43 @@
 {#
-  Copyright (C) 2020 CERN.
+  Copyright (C) 2022 CERN.
 
   Invenio App RDM is free software; you can redistribute it and/or modify it
   under the terms of the MIT License; see LICENSE file for more details.
 #}
 
 {% extends admin_base_template %}
+
+
+{% block admin_page_content %}
+
+<div class="ui container dashboard">
+  <div class="two column stackable ui grid rel-mt-5 computer only tablet only">
```

Comment:
No need to add several times the only class.

Reply:
```suggestion
  <div class="two column stackable ui grid rel-mt-5 computer tablet only">
```
You are right, I tested it locally and it works, thanks!